packages/atlas/src/views/global/NftPurchaseBottomDrawer/NftPurchaseBottomDrawer.tsx

```diff
@@ -0,0 +1,566 @@
+import { formatDuration } from 'date-fns'
+import React, { useCallback, useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+
+import { useNft } from '@/api/hooks'
+import { Avatar } from '@/components/Avatar'
+import { Information } from '@/components/Information'
+import { NftCard } from '@/components/NftCard'
+import { Pill } from '@/components/Pill'
+import { Text } from '@/components/Text'
+import { SvgAlertsWarning24 } from '@/components/_icons'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import { SvgJoystreamLogoShort } from '@/components/_illustrations'
+import { TextField } from '@/components/_inputs/TextField'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { BottomDrawer } from '@/components/_overlays/BottomDrawer'
+import { useBlockTimeEstimation } from '@/hooks/useBlockTimeEstimation'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useMsTimestamp } from '@/hooks/useMsTimestamp'
+import { useSubsribeAccountBalance } from '@/hooks/useSubsribeAccountBalance'
+import { useAsset, useMemberAvatar } from '@/providers/assets'
+import { useJoystream, useTokenPrice } from '@/providers/joystream'
+import { useNftActions } from '@/providers/nftActions'
+import { useSnackbar } from '@/providers/snackbars'
+import { useTransaction } from '@/providers/transactionManager'
+import { useUser } from '@/providers/user'
+import { cVar } from '@/styles'
+import { formatDurationShort } from '@/utils/time'
+
+import {
+  ActionBarCell,
+  ActiveBidWrapper,
+  BidderName,
+  BuyNowInfo,
+  Content,
+  CurrentBidAvatar,
+  CurrentBidJoyToken,
+  CurrentBidWrapper,
+  Divider,
+  EndingTime,
+  FlexWrapper,
+  Header,
+  InnerContainer,
+  Messages,
+  MinimumBid,
+  MinimumBidWrapper,
+  NftPreview,
+  PaymentSplitValues,
+  PaymentSplitWrapper,
+  PlaceBidWrapper,
+  Row,
+  Timer,
+} from './NftPurchaseBottomDrawer.styles'
+
+const TRANSACTION_FEE = 0
+const PLATFORM_ROYALTY = 0
```

Comment:
This will be `1` for now but we should also get this from the chain constant let's add `TODO`